<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>462</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Scrambler</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);
</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);
border: 2px solid #8f8f91;
order-radius: 6px;
border-color: rgb(100, 100, 100); 

</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadDataButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>310</width>
      <height>120</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(40, 40, 40);
    min-width: 80px;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    background-color: rgb(70, 70, 70);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: rgb(100, 100, 100); 
}


</string>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ScrambleButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>310</width>
      <height>120</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(40, 40, 40);
    min-width: 80px;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    background-color: rgb(70, 70, 70);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: rgb(100, 100, 100); 
}


</string>
    </property>
    <property name="text">
     <string>Scramble</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SendDataButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>310</width>
      <height>120</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(40, 40, 40);
    min-width: 80px;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    background-color: rgb(70, 70, 70);
}

QPushButton:flat {
    border: none; 
}

QPushButton:default {
    border-color: rgb(100, 100, 100); 
}


</string>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);
</string>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);
</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);
</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
